Cargo.toml

```diff
@@ -67,64 +67,66 @@ members = [
 clap = { version = "4.0.9", features = ["derive"] }
 codec = { package = "parity-scale-codec", version = "3.0", default-features = false }
 hex-literal = "0.2.1"
-jsonrpsee = { version = "0.15.1", features = ["server", "macros"] }
+jsonrpsee = { version = "0.16.2", features = ["server", "macros"] }
```

Comment:
Did Substrate also upgrade?

Reply:
This is a `paritytech/` crate and it has been updated to `0.16.2`: https://crates.io/crates/jsonrpsee. Polkadot therefore also updated and a few other parachain projects I checked as well (such as Moonbeam)